src/components/Profile/CardList/ProfileCardList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+import Card from "src/components/Profile/CardList/Card";
+import { IGetCardDataContent, IImage, ITheme } from "src/pages/profile/index";
```

Comment:
```suggestion
import { IGetCardDataContent, IImage, ITheme } from "src/pages/profile";
```

next.js 에서는 index 안 붙여도 돼요!

Reply:
아하 감사합니다~!